RIOT-OS/RIOT
# dist/tools/suit: add support for password protected private keys

### Contribution description

This adds support for encrypting private SUIT keys with a password.
The password can be set with the `SUIT_SEC_PASSWORD` variable.

Old (unencrypted) keys are still supported, but new keys will be generated with a default password.


### Testing procedure

Run `suit/publish` in `examples/suit_update`.
This should work both with existing (unenctypted) and new (encrypted) keys.


### Issues/PRs references
